modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/snapshot/SnapshotRestoreCommandOption.java

```diff
@@ -37,7 +37,10 @@ public enum SnapshotRestoreCommandOption implements CommandArg {
 
     /** Synchronous execution flag. */
     SYNC(SnapshotCreateCommandOption.SYNC.argName(), SnapshotCreateCommandOption.SYNC.arg(),
-        SnapshotCreateCommandOption.SYNC.description());
+        SnapshotCreateCommandOption.SYNC.description()),
+
+    /** Check snapshot before restore. */
+    CHECK("--check", null, "Check snapshot before restore(slow!).");
```

Comment:
Let's add note that this check is similar to the `check` command. Smth like that:

Check snapshot **data integrity** before restore (slow!). Similar to the "check" command.